utils/create.cc

```diff
@@ -222,14 +222,17 @@ int tr_main(int argc, char* argv[])
     }
 
     fmt::print(
-        ngettext("{:d} files, {:s}\n", "{:d} file, {:s}\n", builder.fileCount()),
-        builder.fileCount(),
-        tr_formatter_size_B(builder.totalSize()));
+        ngettext("{file_count:d} files, {total_size:s}\n", "{file_count:d} file, {total_size:s}\n", builder.fileCount()),
```

Comment:
No any other place seems to specify types for named placeholders / in translatable strings, except for `L`.

Reply:
good catch. updated 75217ec06488993a39e80b5b686b1cc0a0510d5d